dashboard/modules/node/node_head.py

```diff
@@ -146,7 +161,31 @@ async def _update_nodes(self):
             except Exception:
                 logger.exception("Error updating nodes.")
             finally:
-                await asyncio.sleep(node_consts.UPDATE_NODES_INTERVAL_SECONDS)
+                self._node_update_cnt += 1
+                # Until the head node is registered, we update the
+                # node status more frequently.
+                # If the head node is not updated after 10 seconds, it just stops
```

Comment:
Should there be some sort of logging for better visbility if head node is not updated after 10secs?